### ISSUE TITLE
Website: the docs are showing a banner to go to the latest stable version ON THE LATEST STABLE VERSION

### ISSUE THREAD

**Issue by jean** [argoproj/argo-cd#19333]
Checklist:

* [x ] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [ x] I've included steps to reproduce the bug.
* [ ] I've pasted the output of `argocd version`.

**Describe the bug**

The docs are showing a banner to go to the latest stable version ON THE LATEST STABLE VERSION

**To Reproduce**

Browse to https://argo-cd.readthedocs.io/en/stable/

**Expected behavior**

Don't show the banner on `/stable`

**Screenshots**

![prent](https://github.com/user-attachments/assets/a36d3875-6c11-4500-9ac8-e5f2278c17ab)

**Version**

```shell
Paste the output from `argocd version` here.
```

**Logs**

```
Paste any relevant application logs here.
```

**Comment by Junseokee** [comment]
Hello,
I've solved this problem
But, i have mistakenly made changes directly to the master branch and created a pull request https://github.com/argoproj/argo-cd/pull/19334. To follow best practices and maintain repository integrity, I realize that my changes should have been made in a separate feature branch.

Could you please guide me on the best way to reassign my current PR to a new branch? Alternatively, if there's no way to reassign, should I close the current PR and submit a new one from the correct branch?

**Comment by Junseokee** [comment]
Hello,
I've solved this problem https://github.com/argoproj/argo-cd/pull/19334.
I modified versions.js

**Comment by todaywasawesome** [comment]
Should be fixed with #19334 merge
